client/src/pages/demo/components/index.tsx

```diff
@@ -159,7 +160,18 @@ const DemoComponent: React.FunctionComponent = () => {
             </div>
           </div>
         </div>
-
+        <br />
+        <div className="p-4 border-b border-black space-y-4">
```

Comment:
- Add space between border and -b (your border not applied since it is concatenated)
- Also add an id for easy navigation as per sir Joshua

```suggestion
        <div className="p-4 border -b border-black space-y-4" id="searchbar">
```